geometry/frame.hpp

```diff
@@ -16,12 +16,41 @@ namespace internal_frame {
 using base::not_constructible;
 using base::not_null;
 
-template<typename FrameTag, FrameTag frame_tag, bool frame_is_inertial>
+enum Inertia {
```

Comment:
`ReferenceFrameInertia`; we don't see that name much because it's an `enum` rather than an `enum class`, so it doesn't need to  be catchy, and using up `Inertia` saddens me (see my comment about the possibility of renaming `InertiaTensor` in a recent pull request).

Even better, `ReferenceFrameMotion`: I was thinking of distinguishing rotating and non-rotating inertial frames at some point for `KeplerOrbit`.

Reply:
The class is called `Frame`, not `ReferenceFrame`, so the word "reference" is weird.  Going for `FrameMotion`.